content/Software_Projects/Mobile_SDK/WebAuthn_iOS_Demo_Walkthrough.adoc

```diff
@@ -0,0 +1,256 @@
+== *WebAuthn Walkthrough for iOS*
+
+This is a walkthrough of the iOS client-side implementation of WebAuthn
+using the Yubico https://github.com/Yubico/yubikit-ios/tree/master/YubiKitDemo/YubiKitDemo[YubiKit Demo] included with the https://github.com/Yubico/yubikit-ios[Yubico iOS SDK].
+
+There are really two parts to WebAuthn, *1.* The communication between
+the client (native iOS application or web browser) and the server as
+described by the https://www.w3.org/TR/webauthn[WebAuthn
+specifications] and *2.* The communication between the client (native
+iOS application or web browser) and the authenticator (e.g. YubiKey 5Ci)
+as described by the
+https://fidoalliance.org/specs/fido-v2.0-id-20180227/fido-client-to-authenticator-protocol-v2.0-id-20180227.html[CTAP2 protocol] (Client-To Authenticator Protocol version 2).
+
+This article provides a walkthrough of the communication between a
+native iOS app and a YubiKey 5Ci key as our authenticator plugged into
+an iPhone via the Apple Lightning connector.
+
+== *WebAuthn*
+
+The Web Authentication API (WebAuthn) enables strong first factor (passwordless)
+and multi-factor authentication built on public key cryptography using
+hardware devices like security keys, mobile phones, and other devices
+with built-in authenticators. Learn more https://developers.com/WebAuthn[here].
```

Comment:
developers.com ?

Reply:
Ah, yes. Sorry. That URL is supposed to be https://developers.yubico.com/WebAuthn.